### DISCUSSION TITLE
Issue with HP62231A OfficeJet Pro9010 Printer

### DISCUSSION THREAD

**Discussion by dacomboman** [SpiralLinux/SpiralLinux-project#397] [General]
Have tried many times to print duplex mode with SpiralLinux KDE iteration, and one side of the sheet always prints upside down.
Checked and re-checked all the options out there, rebooted etc. and there was no way for it to do as asked. 

Cinnamon edition does not have this problem.

Any suggestions?

(And no, not sticking to Cinnamon, 'cause i keep missing KDE).

**Comment by geckolinux** [comment]
Hi there, this sounds like an issue to ask about over on the KDE forums.

**Reply by dacomboman** [reply]
Can try.

**Comment by dacomboman** [comment]
From what i have found thus far, is the fact that HP toolbox already pre-installed, automatically installs this printer but remains without a driver.

So, while leaving existing printer as is, went through adding it via HP toolbox again and it seems to be a go ahead this time.

Weird having two HP OfficeJet printers tho.
https://snipboard.io/FxtdWL.jpg

- - - - -

p.s. the laserjet one at the top of the list, was first one installed manually.
